broker/src/main/java/io/camunda/zeebe/broker/system/configuration/backup/S3BackupStoreConfig.java

```diff
@@ -17,6 +18,7 @@ public class S3BackupStoreConfig implements ConfigurationEntry {
   private String region;
   private String accessKey;
   private String secretKey;
+  private Duration apiCallTimeout;
```

Comment:
:wrench: I suggest to provide a default value.

Reply:
I've set it to 180 seconds. If we assume that the largest file of a backup is a 128MiB segment this requires less than 1MiB/s throughput which should be reasonable.